fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/citus/CitusResourceReferenceDAO.java

```diff
@@ -1,157 +0,0 @@
-/*
- * (C) Copyright IBM Corp. 2022
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package com.ibm.fhir.persistence.jdbc.citus;
-
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-
-import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
-import com.ibm.fhir.persistence.jdbc.dao.api.ICommonTokenValuesCache;
-import com.ibm.fhir.persistence.jdbc.dao.api.ILogicalResourceIdentCache;
-import com.ibm.fhir.persistence.jdbc.dao.api.INameIdCache;
-import com.ibm.fhir.persistence.jdbc.dao.impl.ResourceReferenceDAO;
-import com.ibm.fhir.persistence.jdbc.dto.CommonTokenValue;
-import com.ibm.fhir.persistence.jdbc.postgres.PostgresResourceReferenceDAO;
-import com.ibm.fhir.persistence.params.api.ParamSchemaConstants;
-
-/**
- * Citus-specific extension of the {@link ResourceReferenceDAO} to work around
- * some Citus distribution limitations
- */
-public class CitusResourceReferenceDAO extends PostgresResourceReferenceDAO {
```

Comment:
Was this one moved somewhere or we really didn't need it?

Reply:
looks like ResourceReferenceDAO was retired in favor of the CommonValuesDAO, so all ResourceReferenceDAO instances were removed